packages/block-editor/src/components/inserter/menu.js

```diff
@@ -404,7 +404,11 @@ export class InserterMenu extends Component {
 								</div>
 								<Tip>
 									{ __(
-										'While writing, you can press "/" to quickly insert new blocks.'
+										'While writing, you can press'
+									) }
+									<kbd className="edit-post-keyboard-shortcut-help__shortcut-key">/</kbd>
+									{ __(
+										' to quickly insert new blocks.'
```

Comment:
I'm not completely sure of the best way to internationalise a string that has an HTML tag in it. Maybe using `sprintf`. I couldn't see any examples in the codebase. I'll ask for some feedback on that.

Reply:
Feedback on slack is here: https://wordpress.slack.com/archives/C02QB2JS7/p1566976018008100 (requires an account)

Seems as though splitting the string isn't a good idea, but there's no solution yet to internationalising text that has an html element in it. https://github.com/WordPress/gutenberg/pull/16374 is a PR that attempts to address it.

Reply:
Yes. it's very unfortunate that we can't use HTML tags inside translations as of today :(

@nerrad - how far did you get with your implementation for Interpolate component?

Reply:
Unfortunately I think it is a blocker. 😞 

Unless there's a workaround I think it'll hold up the PR.

Reply:
Sorry for the delay here:

> how far did you get with your implementation for Interpolate component?

In the pull I did, the more verbose `createInterpolateElement` can be extracted from that pull and released as is.  It's usable and will solve the problem.  I'm mostly waiting on whether that's a path we want to take initially.

However, the more intuitive jsx-like option with a `<Translate>` or `<Interpolate/>` component is not available yet (maybe could be done on a separate pull?).

Reply:
I opened https://github.com/WordPress/gutenberg/pull/18623 to add the same change but using the newly introduced API method.